crypto/x509/x509_req.c

```diff
@@ -236,6 +236,10 @@ int X509_REQ_add1_attr_by_NID(X509_REQ *req,
                               int nid, int type,
                               const unsigned char *bytes, int len)
 {
+	if (req == NULL || bytes == NULL) {
+            ERR_raise(ERR_LIB_X509, ERR_R_PASSED_NULL_PARAMETER);
+            return 0;
+	}
```

Comment:
The indentation is still wrong for the whole if here.

Reply:
Oops, fixed, and simplified the function.